components/search/SearchFacets.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <b-card
+    v-if="optionsType && optionsType['TYPE']"
```

Comment:
Perhaps just iterate over each of the values inside options and then have a separate component to render the facet options.

Reply:
Every facet _should_ be displayable as checkboxes, even if some down the line may be better displayed using another component.